Features/foutafhandeling.feature

```diff
@@ -32,7 +32,9 @@ We kennen de volgende foutsituaties:
 
 Wanneer de fout is veroorzaakt door fouten in requestparameters (of request body), wordt "invalid-params" gevuld met details over elke foute parameter.
 
-Wanneer er fouten worden veroorzaak door meerdere parameters, dan wordt er per validatiefout een "invalid-params" instantie opgenomen in het antwoord. Alle fouten worden dus teruggegeven.
+Wanneer een parameterwaarde niet overeenkomt met het gespecificeerde type (integer, number, boolean of enumeratie), wordt alleen deze fout gemeld.
+
+Wanneer er meerdere parameterwaarden niet voldoen aan de gespecificeerde eisen (minimum ,maximum, pattern, maxItems, fields, expand) worden al deze fouten tegelijk gemeld
```

Comment:
Hier zou je 'worden al deze fouten tegelijk gemeld' kunnen vervangen door: 'dan wordt voor iedere validatiefout een "invalid parameter" instantie opgenomen in het antwoord.